lib/phoenix/channel/server.ex

```diff
@@ -183,13 +218,14 @@ defmodule Phoenix.Channel.Server do
 
   @doc false
   def init({socket, auth_payload, parent, ref}) do
+    _ = Process.monitor(socket.transport_pid)
```

Comment:
Is the match here deliberate?

Reply:
Yes, we usually do it to say "i know this returns something meaningful, but i am sure i don't need it".